tests/integration_tests/test_upgrade.py

```diff
@@ -143,15 +143,27 @@ def test_clean_boot_of_upgraded_package(session_cloud: IntegrationCloud):
                 assert post_json["v1"]["datasource"].startswith(
                     "DataSourceAzure"
                 )
-        if PLATFORM in ["gce", "qemu"] and CURRENT_RELEASE < NOBLE:
+        if any(
+            [
+                CURRENT_RELEASE > FOCAL,
+                PLATFORM in ["gce", "qemu"],
+            ]
+        ):
```

Comment:
I don't think we want to conflate these two ideas or conditions as we then have to specialize the gce/qemu behavior again and the condition we are looking for related to netplan API is actually only present in `CURRENT_RELEASE >= MANTIC` Since this GCE/QEMU condition is ignoring the comments in 50-cloud-init.yaml due to yaml.load, we can leave this as is.